connect/runtime/src/main/java/org/apache/kafka/connect/util/TopicAdmin.java

```diff
@@ -229,13 +230,18 @@ public boolean createTopic(NewTopic topic) {
                 newlyCreatedTopicNames.add(topic);
             } catch (ExecutionException e) {
                 Throwable cause = e.getCause();
-                if (e.getCause() instanceof TopicExistsException) {
+                if (cause instanceof TopicExistsException) {
                     log.debug("Found existing topic '{}' on the brokers at {}", topic, bootstrapServers);
                     continue;
                 }
                 if (cause instanceof UnsupportedVersionException) {
                     log.debug("Unable to use Kafka admin client to create topic descriptions for '{}' using the brokers at {}," +
-                                      "falling back to assume topic(s) exist or will be auto-created by the broker", topicNameList, bootstrapServers);
+                            "falling back to assume topic(s) exist or will be auto-created by the broker", topicNameList, bootstrapServers);
```

Comment:
Sorry to ask since it is a little orthogonal, but I feel this message is a little vague. Could we just say that the broker does not support the CreateTopics API?

Reply:
Done.